Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.ipp

```diff
@@ -66,9 +66,8 @@ Acts::AdaptiveMultiVertexFitter<input_track_t, linearizer_t>::fitImpl(
       if (perpDist > m_cfg.maxDistToLinPoint) {
         // Relinearization needed, distance too big
         currentVtxInfo.relinearize = true;
-        // Prepare for fit with new vertex position
-        prepareVertexForFit(state, currentVtx, vertexingOptions);
```

Comment:
Is it obvious why this is not needed anymore? Do we do it at a later point for the vertex in question?

Reply:
Yes, it's exactly where you spotted it later during your review! Sorry should have maybe added a comment to the PR!